### ISSUE TITLE
Summary won't output labels of field inside group

### ISSUE THREAD

**Issue by watarutmnh** [verbb/formie#731]
**Description**
Summary won't output labels of field inside group, but outputs field handles instead.

**Steps to reproduce**
1. Add some field inside group field.
2. Add Summary field to form.
3. Check at frontend

### Form settings
- Multi-page form: Yes
- Submission Method: Page Reload
- Client-side Validation: No
- Custom Form Templates: No

**Additional info**
- Plugin version: 1.5.7
- Craft version: 3.7.30.1
- Multi-site: Yes

**Comment by engram-design** [comment]
Fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.5.7",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.5.8](https://github.com/verbb/formie/releases/tag/1.5.8)
